Kernel/Arch/aarch64/Processor.h

```diff
@@ -36,18 +35,19 @@ struct [[gnu::aligned(16)]] FPUState
 extern Processor* g_current_processor;
 
 class Processor {
+    void* m_processor_specific_data[(size_t)ProcessorSpecificDataID::__Count];
```

Comment:
static_cast pls

Reply:
Copy pasted from the x86 version. Will fix!